lib/ansible/module_utils/vmware.py

```diff
@@ -1110,3 +1099,32 @@ def find_datastore_cluster_by_name(self, datastore_cluster_name):
             if dsc.name == datastore_cluster_name:
                 return dsc
         return None
+
+    @staticmethod
+    def wait_for_task(task, max_backoff=64, timeout=3600):
+        """
+        Function to wait for given task using exponential backoff algorithm
```

Comment:
Could you please use PEP 257 to write docstrings?

Reply:
OK.